### PR TITLE
Fixes #15 : JSDoc config, template and readme

### LINKED ISSUE #15: Readme.dist.MD

### PR DESCRIPTION
**Pull request by Omegadela**

### CODE REVIEW

File: README.dist.MD [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The description of the library.
+
+## Installation and Usage
+
+### Installation
+
+To install all dependencies run
+
+```bash
+npm install yarn -g && yarn
```
Comment (ryuran) [inline README.dist.MD]: `npm install library --save`

Comment (ryuran) [reply README.dist.MD]: if library is registered on npm

File: README.dist.MD [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Describe the usage of the library.
+
+```javascript
+// Add some example code
+var library = require('library');
+
+library.foo();
+```
+
+## Dev tools
```
Comment (ryuran) [inline README.dist.MD]: It's only for contributing not for use

**Comment by ghost** [comment]
Your pull request doesn't follow our guidelines. Please fix the following:
- Pull request description must include a screenshot [(?)](https://gitmagic.io/rules/#/pull-request/body-must-include-screenshot)
- Pull request description must include verification steps [(?)](https://gitmagic.io/rules/#/pull-request/body-must-include-verification-steps)
- Pull request description cannot be empty [(?)](https://gitmagic.io/rules/#/pull-request/body-cannot-be-empty)
- Pull request title must be in imperative, present tense (e.g. "add", "fix", "change") [(?)](https://gitmagic.io/rules/#/pull-request/subject-must-be-in-tense)

[Click here](https://app.gitmagic.io/ryuran/jsBoiler/17/ad5cc3ea-3482-41fc-8bbd-4d4876c6f80f/759914c8-a96c-46cf-b68b-8210ab697ee1) for details.

Thank you! :pray:
<sub>
This comment was made by [GitMagic](https://gitmagic.io/) – Magically enforcing your contribution guidelines.
</sub>

**Pull request merged**